### DISCUSSION TITLE
Not able to embed superset in iframe

### DISCUSSION THREAD

**Discussion by malikahsan13** [apache/superset#23373] [Q&A / Help]
I have a php script for generating guest token with following calls

1) api/v1/security/login
2) api/v1/security/csrf_token
3) api/v1/security/guest_token

I am trying to embed supserset dashboard using following script

<script>
        async function fetchGuestTokenFromBackend() {
            let response = await fetch('/superset-login.php');
            let data = await response.json()
            return data.token
        }

            const myDashboard = supersetEmbeddedSdk.embedDashboard({
              id: "57s4581a3-1a61-4284-85d2-200731ff43dsf",
              supersetDomain: "http://localhost:8080",
              mountPoint: document.getElementById("dashboard-container"),
              fetchGuestToken: () => fetchGuestTokenFromBackend(),
              dashboardUiConfig: {
                hideTitle: true,
                hideChartControls: true
              },
            });
      </script>
      
      i am stuck for almost like 2 weeks and i am getting this error
      
      
![image](https://user-images.githubusercontent.com/44003944/225108201-ec900195-c26e-466b-876d-ec4559814b35.png)

my superset_config.py settings are following

SESSION_COOKIE_SAMESITE = None
ENABLE_PROXY_FIX = True
PUBLIC_ROLE_LIKE_GAMMA = True
WTF_CSRF_ENABLED = False
PUBLIC_ROLE_LIKE = "Gamma"
ENABLE_CORS = True
FEATURE_FLAGS = {
    "EMBEDDED_SUPERSET": True
}

CORS_OPTIONS = {
  'supports_credentials': True,
  'allow_headers': ['*'],
  'resources':['*'],
  'origins': ['*']
}

Any help will be appreciated. Thanks in advance !!!

**Comment by harry-pham-viact** [comment]
This answer may help you [link](https://github.com/apache/superset/issues/22258#issuecomment-1333387931)

**Comment by hzdx943** [comment]
I have encountered the same problem. Have you successfully solved the problem？
